package.json

```diff
@@ -111,6 +111,6 @@
     "node": "^4.5 || 6.* || >= 7.*"
   },
   "resolutions": {
-    "nypr-ui": "~0.1.0"
+    "nypr-ui": "nypublicradio/nypr-ui#465e5af"
```

Comment:
Should this be using the versioned nypr-ui? Does nypr-ui need to be updated for that to work?

Reply:
ah, yes! I need to version `nypr-ui`. I'll do that and fix the lockfile before merging

Reply:
FYI just waiting on [this ticket about gradients](https://jira.wnyc.org/browse/RT-398) to make its way to needs merge and I'll bump nypr-ui